### PR TITLE
Installing Liferay DXP on Premises

### PR DESCRIPTION
**Pull request by jhinkey**
@jrhoun 

Here is the content for on-prem install. I replaces the former installation overview articles, and focuses on installing DXP using a Tomcat Bundle.

https://github.com/jhinkey/liferay-learn/blob/installing-liferay-dxp-on-premises/docs/dxp/7.x/en/advanced-installation-and-upgrades/01-installing-liferay-dxp-on-premises/installing-a-liferay-dxp-tomcat-bundle.md

Thanks!
Jim

### CODE REVIEW

File: docs/dxp/7.x/en/advanced-installation-and-upgrades/01-installing-liferay-dxp-on-premises/configuring-a-database.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+However, if your organization requires limiting Liferay DXP database user permissions once the database is initialized, see [High Security Database User Practices](../02-setting-up-liferay-dxp/high-security-database-user-practices.md).
+
+### Configure the Query Result Sort Order (Optional)
+
+Every database has a default order for sorting results (see [this article](https://help.liferay.com/hc/en-us/articles/360029315971-Sort-Order-Changed-with-a-Different-Database)). If you're concerned about this order, consult the database vendor documentation to learn its sort order and if necessary, configure the database to use a default query result order you prefer for Liferay DXP entities.
+
+You've configured your database server, database, and database user. You're ready to install a JDBC connecter that Liferay DXP uses to communicate with the database.
+
```
Comment (jrhoun) [inline docs/dxp/7.x/en/advanced-installation-and-upgrades/01-installing-liferay-dxp-on-premises/configuring-a-database.md]: Hey Jim - 

This section I think adds a lot text that raised a flag for me in review. I wonder if this should be abstracted out to a separate article. Maybe split the three database server configurations away into another article and leave the "Create a blank database w/ utf-8 support" header in. Not sure, just some initial thoughts as I'm reviewing.

File: docs/dxp/7.x/en/advanced-installation-and-upgrades/README.rst [hunk 2]
```diff
@@ -1,29 +1,23 @@
...
 
--  :doc:`/advanced-installation-and-upgrades/01-installing-liferay-dxp/01-getting-started-with-liferay-dxp`
--  :doc:`/advanced-installation-and-upgrades/01-installing-liferay-dxp/02-installation-overview`
--  :doc:`/advanced-installation-and-upgrades/01-installing-liferay-dxp/03-obtaining-liferay`
--  :doc:`/advanced-installation-and-upgrades/01-installing-liferay-dxp/04-connecting-a-database`
--  :doc:`/advanced-installation-and-upgrades/01-installing-liferay-dxp/05-using-the-setup-wizard`
+-  :doc:`/advanced-installation-and-upgrades/01-installing-liferay-dxp-on-premises/installing-a-liferay-dxp-tomcat-bundle`
+-  :doc:`/advanced-installation-and-upgrades/01-installing-liferay-dxp-on-premises/configuring-a-database`
+-  :doc:`/advanced-installation-and-upgrades/01-installing-liferay-dxp-on-premises/running-liferay-dxp-for-the-first-time`
 -  `Using DXP Cloud <https://learn.liferay.com/dxp-cloud-latest/using-the-liferay-dxp-service/introduction-to-the-liferay-dxp-service.html>`__
```
Comment (jrhoun) [inline docs/dxp/7.x/en/advanced-installation-and-upgrades/README.rst]: This article suddenly feels out of place in the On Premises section. Yet we also want to pivot our focus to encouraging users to check out DXP Cloud. @jhinkey how do you think we should present the DXP Cloud option in this context?

Comment (jhinkey) [reply docs/dxp/7.x/en/advanced-installation-and-upgrades/README.rst]: @jrhoun I added a placeholder for Hosting Liferay DXP. Let's present the DXP Cloud option, along with Docker image and on-prem in that article.

File: docs/dxp/7.x/en/advanced-installation-and-upgrades/01-installing-liferay-dxp-on-premises/configuring-a-database.md (hunk 1 above)
Comment (jhinkey) [reply docs/dxp/7.x/en/advanced-installation-and-upgrades/01-installing-liferay-dxp-on-premises/configuring-a-database.md]: @jrhoun I'm thinking about this. As sys admins are reading this article, we have to assume they're setting up a server for non-trivial purposes. They at least need to know what the steps are for creating a production-grade database. They can discern what level of detail to go through depending on their use case. But I'll consider how we might shorten this and possibly extract some details to a reference doc.

Comment (jhinkey) [reply docs/dxp/7.x/en/advanced-installation-and-upgrades/01-installing-liferay-dxp-on-premises/configuring-a-database.md]: @jrhoun I force pushed a commit that refines Configuring a Database to 4  simple steps.

**Comment by jhinkey** [comment]
Closing this in light of the rebased PRs

**Pull request closed without merging**